lib/abstract_format.ml

```diff
@@ -688,6 +696,18 @@ and cls_of_sf ?(in_function=false) sf : (clause_t, err_t) Result.t =
      let%bind body = sf_body |> expr_of_sf |> track ~loc:[%here] in
      ClsCase {line; pattern; guard_sequence = Some guard_sequence; body} |> return
 
+  (* if clause Gs -> B *)
+  | Sf.Tuple (5, [
+                 Sf.Atom "clause";
+                 Sf.Integer line;
+                 Sf.List [];
```

Comment:
I have confirmed that the 3rd element of the other non-function clauses isn't empty list. So it's OK.